crates/core/src/factories/mem_peer_meta_store/test.rs

```diff
@@ -39,6 +39,18 @@ impl TestPeerMetaStore {
 
         Ok(())
     }
+
+    async fn put_unresponsive_peer(
```

Comment:
Is there a reason not to call this also `mark_peer_unresponsive`? I think I would find it more intuitive.

Reply:
Slightly different suggestion. The `inner` field is visible at the module level so could actually do without these methods and just use `store.inner.mark_peer_unresponsive`?